avocado/utils/cpu.py

```diff
@@ -550,6 +550,23 @@ def wrap(*args, **kwargs):
     return wrap
 
 
+def lscpu():
+    # Getting the values from lscpu
+    output = process.run("lscpu")
+    # res list contains values as : Cores , Physical Sockets , Physical Chips , Chips
+    res = []
```

Comment:
What about to use dict here, since you want to make it expandable? It will be easier to add new values without changing the return value.